### ISSUE TITLE
custom character

### ISSUE THREAD

**Issue by cjz37** [kbrodt/sketch2pose#8]
Does this project support direct generation of custom character?
In the paper, you used Blender to transfer.But I didn't find a similar plugin. Did you do skeletal binding first?

**Comment by kbrodt** [comment]
No, it only predicts the pose of SMPL model. For ratergeting I used the https://github.com/Mwni/blender-animation-retargeting

**Comment by cjz37** [comment]
Didn't that plugin require the model to have a skeleton? SMPL model seems to have no skeleton

**Comment by kbrodt** [comment]
Download SMPL plugin for blender from https://smpl-x.is.tue.mpg.de/ which has skeleton

**Comment by cjz37** [comment]
I see. Thank you very much for your answer！！
